akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Protocol.scala

```diff
@@ -17,6 +17,8 @@ object Http2Protocol {
   val InitialConnectionLevelWindow = 65535
   val InitialStreamLevelWindow = 65535
 
+  val NoStreamId = 0
```

Comment:
This is not a constant I invented but it's the one from the spec. But, yes, I should add a comment.  :)